.github/workflows/ci-build.yml

```diff
@@ -22,21 +50,15 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
     
-    - name: Cache dependencies
-      uses: actions/cache@v2
-      with:
-        path: ~/go/pkg/mod
-        key: ${{ runner.os }}-go-${{ hashFiles ('**/go.sum') }}
-        restore-keys: |
-          ${{ runner.os }}-go-
```

Comment:
I forgot to ask in the other PR - why do you remove this step?

Reply:
because it causes troubles (ie, lots of error messages) when used in conjunction with the golangci/golangci-lint-action

Reply:
actually, this may have been fixed now. I will give it a try later to see if the `Cannot open: File exist` errors disppeared